examples/preload/test/utils.ts

```diff
@@ -24,6 +24,7 @@ export const startServerNewProcess = async ({
         // This is retquired in this set of tests because it validates changes to globals
         nodeEnvironmentsMode: 'forked',
         runtimeOptions,
+        featureDiscoveryRoot: 'dist',
```

Comment:
was this needed? afaik the tests of the engine still run from source...? did it not work otherwise?